Joining together CTRP data.Rmd

```diff
@@ -0,0 +1,304 @@
+---
+title: "Joining together CTRP data"
+output: github_document
+---
+
+## 1. Load packages
+```{r load packages}
+
+library(data.table)
+library(readr)
+library(dplyr)
+library(tidyverse)
+library(knitr)
+library(lubridate)
+
+```
+
+## 2. Load data
+```{r load data}
+
+setwd("~/Desktop/GitHub/Homework_5/HW05/CTRP_files")
```

Comment:
That's a good idea to set your working directory to the CTRP_files folder. I defined the path separately inside each read() function, but this method seems to make things much easier.